docs/en/stack/ml/dataframes.asciidoc

```diff
@@ -0,0 +1,57 @@
+[[ml-dataframes]]
+=== {dataframes-cap}
+
+{dataframes-cap} feature is available in 7.2 and later.
```

Comment:
I don't think this availability date information is necessary. This page will only exist in 7.2 and later documentation and this functionality be mentioned in the 7.2 release highlights so that should be sufficient.